### ISSUE TITLE
The character will not grow fat, and frequent calorie warnings are displayed.

### ISSUE THREAD

**Issue by Ezsk** [CleverRaven/Cataclysm-DDA#51536]
### Describe the bug

No matter how much food I eat, my stored calories do not exceed 40,000, and frequently show up as " You feel that you need to eat more calorie-dense food."

I am not sure what exactly is happening. However, I have noticed that the low calorie warning messages are appearing more frequently, and the characters are not gaining fat no matter how much they eat. I think others will need to supplement this.

### Steps To Reproduce

1. Make New Character
2. Set Stored calories to 39000
3. Eat some High calorie foods.
4. Wait a few hours for the food in your intestines to be digested.
5. The character is not fat. and Stored calories do not go over 40,000.

### Expected behavior

Able to get over 40000 calories.
Allows players to get fat.

### Versions and configuration

OS : Windows 10
Game Version : Experimental Version 957defc (released on 2021-09-09-0239)

**Comment by ProfoundDarkness** [comment]
TLDR; could NOT replicate on Ubuntu 20.04 Linux x64, cdda version 0.F-2541-g21435d22a0 with mods dda, no_npc_food.

I thought I'd try this since I've noticed in my current game that it seems like I should have switched to overweight once once (still getting used to the new systems)...  I don't know what appropriate test foods are, I went with cooked fatty meat (kcal 1300, satiety ||||\ ).
For step 1 I hit play now which spawned me as a true food person...  Traits Thick skinned and true foodperson.  No mutations.
So as the 2nd replication step suggested, set stored calories to 39000, this changed Weight to normal in sidebar.
Next step was to eat some food high in calories... Any time satisfied cleared from hunger I ate another helping.  Enough water to clear thirsty.  I did some driving on the first day but almost no walking, teleporting in most cases.
Eventually found a nice quiet place to wait the day away.
I don't know what the threshold to overweight is. but apparently I crossed it while sleeping through the middle of day 2 (couldn't fall asleep, any time satisfied cleared, stopped trying to eat more).
When my test food person woke up, stored kcal shows 59314 at 7pm on day 2 with a state of overweight.

So I thought I'd debug-check my character in my current game, playing on windows (updated about once a day from master, mxe compiled).  kcal is 44863.

Aside: driving a food truck at 80 mph (or so) through fields and roads with starting skill of 0 was... interesting.  Lots of skidding somehow usually in the right enough direction with some panicked hand break pulls.  Ultica is looking neat.

**Comment by Ezsk** [comment]
Thank you for your confirmed, and I am very sorry, this reproduction step does not seem to work. Probably the threshold of 40000 is meaningless.
I' m trying to find the correct reproduction steps now.

It all started with a report in our CDDA community. There were several players who said, "I've been consuming a lot of food lately, and I don't know why, but no matter how much I eat, it's not enough." I feel the same in my current game. I' ve been trying to take in calories with MREs and such every time I get hungry, but I'm getting trouble with the frequent calorie lack warnings.

The actual gameplay may have increased the calorie requirement due to some factors.
Anyway, I will continue to look for the cause. Thank you.

**Comment by actual-nh** [comment]
Any mutations?

**Comment by Ezsk** [comment]
I am sorry but the issue could not be confirmed to be reproducible, so it is closed.
